### ISSUE TITLE
Detect and Close When Client Isn't Running

### ISSUE THREAD

**Issue by Sean-Kurian** [Sean-Kurian/OnlyLs#4]
Currently if you launch while the client isn't running, the executable will throw an error. Catch this exception and handle gracefully instead.

**Comment by cmrngoode** [comment]
I'm getting this error when I try to run the program normally or as an admin while the client is open:

There was an error while processing your request: [Errno 2] No such file or directory: 'C:/Riot Games/League of Legends/lockfile'

**Comment by Sean-Kurian** [comment]
I will look into this today - do you have multiple drives, and is the Riot Games folder in a different one?

**Comment by cmrngoode** [comment]
I just have one drive and the riot folder is in it's default directory. I have the program on my desktop.

**Comment by Sean-Kurian** [comment]
Can you find your riot games installation folder while the client is running? Where is it? And is there a lockfile if you follow the subdirectories? (lockfile will only be present while the client is running)

EDIT: Also, what's your Operating System?

**Comment by cmrngoode** [comment]
yes it's in C:/Riot Games/League of Legends but there's no "lockfile"

I'm running Windows 11 Home.

**Comment by Sean-Kurian** [comment]
Really weird, as far as I know the client has to generate a lockfile when running in that specific subdirectory. Do you want to add me on Discord? Intrinsically#6414

**Comment by retro4hack** [comment]
Hi, I've created a pull request for this issue some days ago. Probably you missed it, let me know :D
